backend/substrapp/tasks/tasks.py

```diff
@@ -36,6 +39,8 @@
 COMPOSITE_TRAINTUPLE_TYPE = 'compositeTraintuple'
 TESTTUPLE_TYPE = 'testtuple'
 
+TAG_VALUE_FOR_TRANSFERT_BUCKET = "sendToTransfertBucket"
```

Comment:
This tag value is maybe to long and error prone with this typo, no ?
I fell like something simpler could be enough like `bucket` :)

Reply:
half-done -> `transfertBucket`. Is it ok for you?